.github/ISSUE_TEMPLATES/bug.md

```diff
@@ -0,0 +1,30 @@
+---
+
+name: 🐛 Bug report  
+about: Create a report to help us improve
+---
+
+## 🐛 Bug Report
+
+A clear and concise description of what the bug is.
+
+## To Reproduce
+
+Steps to reproduce the behavior:
+
+## Expected behavior
+
+A clear and concise description of what you expected to happen.
+
+## Link to repl or repo
+
+Please provide either a [repl.it demo](https://repl.it/languages/jest) or a
+minimal repository on GitHub.
```

Comment:
Maybe add some wording around how it's _highly_ encouraged? Personally I rarely bother with bug reports, unless they are dead simple to reproduce (and having a repo is just that)

Reply:
+1, also mentioned that if it's not included it'll basically stall